### DISCUSSION TITLE
Change port settings rtorrent / disk full but I got 1tb free

### DISCUSSION THREAD

**Discussion by vildsvin666** [swizzin/swizzin#1192] [Q&A + Help]
Hi!
Where can i find the rtorrent conf file so I can switch connection port? Want only one port and not random. When change throu rutorrent it seems not to work. Also rutorrent complains that disk is full but I got 1tb free space. :)

**Comment by siedenburg2** [comment]
There should be a config file in your user dir (/home/username/.rtorrent.rc)

**Reply by vildsvin666** [reply]
Yes found it with nano, thanks :) Works fine now.
